WordPress/Classes/ViewRelated/System/WPTabBarController.h

```diff
@@ -0,0 +1,38 @@
+#import <UIKit/UIKit.h>
+
+extern NSString * const kWPNewPostURLParamContentKey;
+extern NSString * const kWPNewPostURLParamTagsKey;
+
+typedef NS_ENUM(NSUInteger, WPTabType) {
+    WPTabMySites,
+    WPTabReader,
+    WPTabNewPost,
+    WPTabMe,
+    WPTabNotifications
+};
+
+@class AbstractPost, Blog, BlogListViewController, MeViewController , NotificationsViewController, ReaderPostsViewController;
```

Comment:
I'm not sure we have anything in our code style about it, but would it help readability to have these on individual lines?